pkg/cmd/run.go

```diff
@@ -127,6 +132,18 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 				}
 			}
 
+			if addKey && preserveEnv {
+				// skip key if environment variables already contain this key
+				envKey := fmt.Sprintf("%s=", name)
+				for _, envVariable := range env {
+					if strings.HasPrefix(envVariable, envKey) {
```

Comment:
is the prefix because the var `envVariable` now is:
`TOKEN=1234324545345` ?
If that is the case can you strip out the `envVariable` the same way we do `envKey`? It's bit confusing otherwise

Reply:
I've updated the implementation so that it's more straightforward